ros/src/fsds_ros_bridge/include/airsim_ros_wrapper.h

```diff
@@ -87,14 +90,43 @@ class AirsimROSWrapper
     int64_t first_imu_unreal_ts = -1;
 
 private:
+    // STATISTICS objects (where possible,
+    // the convention of RpcCall+Statistics
+    // or Pub/Sub name + _statistics
+    // for naming is followed)
+    // These instances need to be initialized elsewhere than where they are declared (ie not here)
+    ros_bridge::Statistics setCarControlsStatistics;
+    ros_bridge::Statistics getGpsDataStatistics;
+    ros_bridge::Statistics getCarStateStatistics;
+    std::vector<ros_bridge::Statistics> getImuDataVecStatistics;
+    std::vector<ros_bridge::Statistics> simGetImagesVecStatistics;
+    std::vector<ros_bridge::Statistics> getLidarDataVecStatistics;
+    ros_bridge::Statistics control_cmd_sub_statistics;
+    ros_bridge::Statistics global_gps_pub_statistics;
+    ros_bridge::Statistics odom_local_ned_pub_statistics;
+    std::vector<ros_bridge::Statistics> cam_pub_vec_statistics;
+    std::vector<ros_bridge::Statistics> lidar_pub_vec_statistics;
+    std::vector<ros_bridge::Statistics> imu_pub_vec_statistics;
+    
+    // create std::vector<Statistics*> which I can use to iterate over all these options 
+    // and apply common operations such as print, reset
+    // std::vector<ros_bridge::Statistics*> statistics_obj_ptr;

```

Comment:
this was a failed attempt of storing references to all statistics instances in a vector so that I would be able to apply operations to all instances easily. I'm keeping it to use in the future